src/goCover.ts

```diff
@@ -305,3 +309,20 @@ export function toggleCoverageCurrentPackage() {
 		});
 	});
 }
+
+export function isPartOfComment(e: vscode.TextDocumentChangeEvent): boolean {
+	let result = false;
+	e.contentChanges.every(function (change, index) {
+		let line = e.document.lineAt(change.range.start);
```

Comment:
```
return e.contentChanges.every(change => {
// Just return true or false based on your logic here
}
```

> I guess it would be a good idea not to return as soon as we find that the change is a part of a comment. If another change which is not a part of a comment is found in a later iteration, then we would have to anyway remove the highlighting right?

A `return` statement in the callback passed to the `.every` doesnt cause a return from `isPartOfComment`. 
A `return true` statement in the callback passed to the `.every` cause the iteration to continue;
A `return false` statement in the callback passed to the `.every` cause the iteration to stop;

Reply:
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
Thanks for the detailed explanation.